Multiple TXT records in apex causes panic

Having two apex TXT records causes a panic. e.g. the following config:

    TXT('@','google-site-verification=abcd1234'),
    TXT('@','google-site-verification=wxyz9876'),

causes panic:
```
panic: interface conversion: interface {} is nil, not *models.RecordConfig

goroutine 1 [running]:
panic(0xa0e120, 0xc0422be1c0)
        /usr/local/go/src/runtime/panic.go:500 +0x1af
github.com/StackExchange/dnscontrol/providers/namedotcom.(*nameDotCom).GetDomainCorrections(0xc0424d1420, 0xc04226c3f0, 0xc0426f17b0, 0xf, 0xc042380658, 0x1, 0x0)
        /go/src/github.com/StackExchange/dnscontrol/providers/namedotcom/records.go:44 +0x98c
main.main()
        /go/src/github.com/StackExchange/dnscontrol/main.go:184 +0x1051
```

--- a/providers/namedotcom/records.go
+++ b/providers/namedotcom/records.go
@@ -41,7 +41,7 @@ func (n *nameDotCom) GetDomainCorrections(dc *models.DomainConfig) ([]*models.Co
 		corrections = append(corrections, c)
 	}
 	for _, cre := range create {
-		rec := cre.Desired.Original.(*models.RecordConfig)
+		rec := cre.Desired
 		c := &models.Correction{Msg: cre.String(), F: func() error { return n.createRecord(rec, dc.Name) }}
 		corrections = append(corrections, c)
 	}

